src/components/[guild]/activity/ActivityLogFiltersBar/components/FiltersInput.tsx

```diff
@@ -0,0 +1,253 @@
+import {
+  Box,
+  Flex,
+  HStack,
+  IconButton,
+  Input,
+  Stack,
+  Text,
+  useColorModeValue,
+} from "@chakra-ui/react"
+import * as combobox from "@zag-js/combobox"
+import { normalizeProps, useMachine } from "@zag-js/react"
+import { useRouter } from "next/router"
+import { CaretDown, X } from "phosphor-react"
+import { KeyboardEvent, useEffect, useRef, useState } from "react"
+import {
+  isSupportedQueryParam,
+  SupportedQueryParam,
+  useActivityLogFilters,
+} from "./ActivityLogFiltersContext"
+import Dropdown from "./Dropdown"
+import FilterTag from "./FilterTag"
+import Suggestion from "./Suggestion"
+
+const FILTERS_TO_RENDER: SupportedQueryParam[] = [
```

Comment:
Why is this needed instead of just using SUPPORTED_SEARCH_OPTIONS?

Reply:
Good point, fixed in 1f2c67a5892e0ed31913f387a757fbab3c240e0b